### PR TITLE
Additional features

### PR DESCRIPTION
**Pull request by Paramveer12**
Screenshot_20200606_180523
Screenshot_20200606_180552

### CODE REVIEW

**Comment by wayne9499** [comment]
alignment is really bad

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@angular/platform-browser": "~9.1.9",
     "@angular/platform-browser-dynamic": "~9.1.9",
     "@angular/router": "~9.1.9",
+    "bcrypt": "^4.0.1",
```
Comment (lakshay-angrish) [inline package.json:24]: why do you need this?

Comment (Paramveer12) [reply package.json:24]: i don't know, i just installed it

Comment (lakshay-angrish) [reply package.json:24]: well, remove this line then

**Comment by lakshay-angrish** [comment]
> alignment is really bad
@Paramveer12 Did you fix this? You could use an angular material card.

**Comment by Paramveer12** [comment]
Will do it next time

**Pull request merged**